tests/test_enum/bad/output/useless_wildcard.catala_en.A.out

```diff
@@ -0,0 +1,7 @@
+[ERROR] Unreachable match case, all constructors of the enumeration E are already specified.
```

Comment:
Ok so for that one you can actually transform it into a warning, since it does not prevent you from compiling the code :)